### PR TITLE
docs-third-party-add-gelato-vrf

### PR DESCRIPTION
**Pull request by anirudhmakhana**
This PR adds Gelato VRF, a trusted decentralized source for real randomness, to the documentation for Arbitrum. Gelato VRF is now available for developers building on Arbitrum, providing genuine random values that can be verified for authenticity.

### CODE REVIEW

**Comment by anirudhmakhana** [comment]
I've signed the CLA but it still adding the error status. @symbolpunk could i get some assitance here? Thanks!

**Comment by hkalodner** [comment]
@cla-bot check

**Comment by hkalodner** [comment]
Sorry about the delay on the CLA!

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+title: "Getting Started with Gelato VRF"
+description: "Learn how to use Gelato VRF"
+author: anirudhmakhana
+sme: anirudhmakhana
+sidebar_label: "Gelato VRF"
+---
+
+# What is Gelato VRF?
+
+Gelato VRF offers real randomness for blockchain applications on Arbitrum by leveraging Drand, a trusted decentralized source for random numbers. With Gelato VRF, developers on Arbitrum get random values that are both genuine and can be checked for authenticity. Explore Gelato VRF's support for all supported networks [here](https://docs.gelato.network/web3-services/vrf/supported-networks).
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:11]:
```suggestion
Gelato VRF offers genuine randomness for blockchain applications on Arbitrum by leveraging Drand, a trusted decentralized source for random numbers. With Gelato VRF, developers on Arbitrum get random values that are both genuine and can be checked for authenticity. Explore Gelato VRF's support for all supported networks [here](https://docs.gelato.network/web3-services/vrf/supported-networks).
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## Applications of Gelato VRF
+
+The potential applications of a reliable and transparent random number generator on the blockchain are vast. Here are just a few use cases:
+
+- Gaming and Gambling: Determine fair outcomes for online games or decentralized gambling applications.
+- Decentralized Finance (DeFi): Use in protocols where random selections, like lottery systems, are required.
+- NFT Generation: Randomly generate traits or characteristics for unique digital assets.
+- Protocol Decision Making: In protocols where decisions need to be randomized, such as selecting validators or jurors.
+
+# How does Gelato VRF Work?
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:22]:
```suggestion
# How does Gelato VRF work?
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+- Drand: Gelato VRF utilizes Drand, a decentralized randomness beacon ensuring unpredictability and unbiased randomness.
+
+Top-level Flow:
+
+- Contract Deployment: Use GelatoVRFConsumerBase.sol as an interface for requesting random numbers.
+- Requesting Randomness: Emit the RequestedRandomness event to signal the need for a random number.
+- Processing: Web3 functions fetch the random number from Drand.
+- Delivery: The fulfillRandomness function delivers the random number to the requesting contract.
+
+# Quick Start Guide
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:37]:
```suggestion
# Quick start guide
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+# Quick start guide
+
+In order to get your VRF up and running with Gelato, you need to make your contract VRF Compatible.
+
+## Step 1: Setup your development environment
+
+Ensure you have either [Foundry](https://book.getfoundry.sh/getting-started/installation) or [Hardhat](https://hardhat.org/) set up in your development environment.
+
+## Step 2: Install the Gelato VRF Contracts
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:45]:
```suggestion
## Step 2: Install the Gelato VRF contracts
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+pragma solidity 0.8.18;
+
+import {GelatoVRFConsumerBase} from "./GelatoVRFConsumerBase.sol";
+
+contract YourContract is GelatoVRFConsumerBase {
+    // Your contract's code goes here
+}
+```
+
+## Step 4: Request Randomness
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:72]:
```suggestion
## Step 4: Request randomness
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        uint64 requestId,
+        bytes memory data,
+    ) internal override {
+    }
+}
+```
+
+## Step 6: Pass dedicated msg.sender
+
+When you're ready to deploy your Gelato VRF-compatible contract, an important step is to include the dedicated msg.sender as a constructor parameter. This ensures that your contract is set up to work with the correct operator for fulfilling the randomness requests.. It's crucial for ensuring that only authorized requests are processed.
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:95]:
```suggestion
When you're ready to deploy your Gelato VRF-compatible contract, an important step is to include the dedicated `msg.sender` as a constructor parameter. This ensures your contract is set up to work with the correct operator to fulfill the randomness requests. It's crucial to ensure that only authorized requests are processed.
```

File: arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    constructor(address operator)
+        GelatoVRFConsumerBase(operator) {
+        // Additional initializations
+    }
+
+    // The rest of your contract code
+}
+```
+
+and once you have your contract ready & deployed, grab the address and [Deploy your VRF instance](https://docs.gelato.network/web3-services/vrf/quick-start/deploying-your-vrf-instance)!
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Gelato/gelato-vrf.md:113]:
```suggestion
Once your contract is ready & deployed, grab the address and [Deploy your VRF instance](https://docs.gelato.network/web3-services/vrf/quick-start/deploying-your-vrf-instance)!
```

**Review by anegg0 — APPROVED** [review]

**Comment by hkalodner** [comment]
@cla-bot check

**Review by anegg0 — APPROVED** [review]
LGTM

**Pull request merged**